### PR TITLE
parallel programming: reworded few sentences

### PR DESCRIPTION
**Pull request by bast**
more changes coming in separate PRs

### CODE REVIEW

File: parallel.rst [hunk 1]
```diff
@@ -4,20 +4,21 @@ Parallel programming
 .. questions::
 
    - When you need more than one processor, what do you do?
-   - What are the limitations of How to use basic NumPy?
+   - How can we use more than one processor/core in Python?
+   - What are the limitations of using basic NumPy?
```
Comment (rkdarst) [inline parallel.rst:8]: I probably wrote this, but wonder if this is covered/a relevant question for this episode?

**Review by rkdarst — APPROVED** [review]
One comment, can merge anyway since I don't think it's a major thing.

File: parallel.rst (hunk 1 above)
Comment (bast) [reply parallel.rst:8]: Yeah I don't think we talk about that aspect in the text.

Comment (rkdarst) [inline parallel.rst:8]:
```suggestion
```

**Pull request merged**